src/Illuminate/Auth/Guard.php

```diff
@@ -375,11 +375,12 @@ public function attempt(array $credentials = [], $remember = false, $login = tru
 		// fact valid we'll log the users into the application and return true.
 		if ($this->hasValidCredentials($user, $credentials))
 		{
-			if ($login) $this->login($user, $remember);
+			if ($login)
 			{
-				return true;
-
+				$this->login($user, $remember);
```

Comment:
@yasinkocak It's meant to be always true. Check this commit before the problem commit: https://github.com/laravel/framework/blob/283a320aa68ec1613661853e6b213fba64da0629/src/Illuminate/Auth/Guard.php#L366-L368